jobserver/views/orgs.py

```diff
@@ -22,16 +25,35 @@ def get_success_url(self):
 
 
 class OrgDetail(DetailView):
-    model = Org
-    slug_url_kwarg = "org_slug"
     template_name = "org_detail.html"
 
-    def get_context_data(self, **kwargs):
-        projects = self.object.projects.order_by("name")
-
-        context = super().get_context_data(**kwargs)
-        context["projects"] = projects
-        return context
+    def get(self, request, *args, **kwargs):
+        slug = self.kwargs["org_slug"]
+
+        try:
+            org = Org.objects.get(slug=slug)
+        except Org.DoesNotExist:
+            # we used to serve up Workspaces at the root URL but have moved
+            # them under their relevant Orgs & Projects.  So attempt to
+            # redirect requests to a valid Workspace here for a while.
+
+            try:
+                workspace = Workspace.objects.get(name=slug)
+            except Workspace.DoesNotExist:
+                raise Http404
```

Comment:
What happens if you get multiple workspaces with the same URL?

Reply:
Workspace names are already unique in the model so we should't do.